source/text/Format.cpp

```diff
@@ -277,7 +279,7 @@ string Format::Credits(int64_t value)
 	int64_t absolute = abs(value);
```

Comment:
There's also a problem here where, if "value" is -2^63 (the most negative value representable by a signed 64 bit integer) the result of the call to abs is still negative (because the most positive value representable by a signed 64 bit integer is 2^63 - 1) so the subsequent checks of if "absolute" exceeds the threshold return false even though it is in fact very big.
Making `absolute` unsigned should probably fix this.
```suggestion
	uint64_t absolute = abs(value);
```

`Format.cpp::<unnamed>::FormatInteger` will also need to be updated to take an unsigned integer (it shouldn't be getting passed negative valeus anyway, it has a separate bool parameter for negativity.)

Reply:
FormatInteger never actually gets called with such a large value, as all usages filter with the threshold.

Reply:
I changed it so we use the floating-point absolute value, which should always have the correct sign.